testkit/src/main/scala/org/bitcoins/eclair/rpc/EclairRpcTestUtil.scala

```diff
@@ -215,8 +226,100 @@ trait EclairRpcTestUtil extends BitcoinSLogger {
     ()
   }
 
+  private def createNodeLink(
+      bitcoindRpcClient: Option[BitcoindRpcClient],
+      channelAmount: MilliSatoshis)(implicit actorSystem: ActorSystem): (
+      EclairRpcClient,
+      EclairRpcClient,
+      EclairRpcClient,
+      EclairRpcClient) = {
+    implicit val ec: ExecutionContext = actorSystem.dispatcher
+    val internalBitcoind = bitcoindRpcClient.getOrElse(
+      BitcoindRpcTestUtil.startedBitcoindRpcClient()
+    )
+
+    val (first, second) = createNodePair(Some(internalBitcoind))
+    val (third, fourth) = createNodePair(Some(internalBitcoind))
+
+    def open(
+        c1: EclairRpcClient,
+        c2: EclairRpcClient): Future[FundedChannelId] = {
+      openChannel(c1,
+                  c2,
+                  channelAmount.toSatoshis,
+                  MilliSatoshis(channelAmount.toLong / 2))
+    }
+
+    EclairRpcTestUtil.connectLNNodes(second, third)
+
+    val openF = Future.sequence(
+      List(open(first, second), open(second, third), open(third, fourth)))
+
+    AsyncUtil.awaitCondition(() => openF.isCompleted)
```

Comment:
nit: 

```scala

val _ = AsyncUtil.awaitCondition(() => openF.isCompleted)

val _ = internalBitcoind.generate(3)

```

Reply:
It's not possible to do multiple `val _ =  foo` in the same block :-( And if we do `val __ =  foo` we get warnings about unused params